tensorboard/plugins/profile/profile_plugin_loader.py

```diff
@@ -49,6 +49,15 @@ def load(self, context):
         Returns:
           A ProfilePlugin instance or None if it couldn't be loaded.
         """
+        try:
+            import tensorboard_plugin_profile
+
+            # If the dynamic plugin is available, don't load this plugin
+            # at all.
+            return None
+        except ImportError:
+            pass
+
```

Comment:
It seems we will need line:61-69 some where in profiler dynamic plugin.
We cannot have dynamic plugin directly depends on tensorflow in pip dependency, as that will cause circular dependency.

Reply:
Yes, agreed.